### PR TITLE
release-1.19: add 1.19.0-rc4 to the release-1.19 schedule

### PR DESCRIPTION
**Pull request by cpanato**
#### What type of PR is this:
/kind documentation

#### What this PR does / why we need it:
Add the 1.19.0-rc4 release cut to the release-1.19 schedule

#### Special notes for your reviewer:

/assign @onlydole

### CODE REVIEW

File: releases/release-1.19/release-notes-draft.md [hunk 1]
```diff
@@ -30,7 +30,7 @@
     plugins implement an `Unreserve` call.
   - `.disablePreemption` was removed. Users can disable preemption by disabling the
     "DefaultPreemption" PostFilter plugin. ([#91420](https://github.com/kubernetes/kubernetes/pull/91420), [@pancernik](https://github.com/pancernik)) [SIG Scheduling]
- 
+
```
Comment (saschagrunert) [inline releases/release-1.19/release-notes-draft.md:33]: Do we wanna revert those spacing changes?

Comment (cpanato) [reply releases/release-1.19/release-notes-draft.md]: done @saschagrunert

File: releases/release-1.19/release_team.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
-| Docs | Savitha Raghunathan ([@savitharaghunathan](https://github.com/savitharaghunathan) / Slack: `@sraghunathan`) | Anna Jung ([@annajung](https://github.com/annajung) / Slack: `@annajung`), Mikael Johansson ([@mikejoh](https://github.com/mikejoh) / Slack: `@mikjoh`), Zachary Estrella ([@zestrells](https://github.com/zestrells) / Slack: `@zestrella`), Divya Mohan ([@divya-mohan0209](https://github.com/divya-mohan0209) / Slack: `@Divya Mohan`) | 
+| Docs | Savitha Raghunathan ([@savitharaghunathan](https://github.com/savitharaghunathan) / Slack: `@sraghunathan`) | Anna Jung ([@annajung](https://github.com/annajung) / Slack: `@annajung`), Mikael Johansson ([@mikejoh](https://github.com/mikejoh) / Slack: `@mikjoh`), Zachary Estrella ([@zestrells](https://github.com/zestrells) / Slack: `@zestrella`), Divya Mohan ([@divya-mohan0209](https://github.com/divya-mohan0209) / Slack: `@Divya Mohan`) |
```
Comment (saschagrunert) [inline releases/release-1.19/release_team.md:9]: Found another space change :grin:

Comment (cpanato) [reply releases/release-1.19/release_team.md:9]: there is an issue to remove the spaces?

Comment (cpanato) [reply releases/release-1.19/release_team.md]: reverted as well @saschagrunert

**Pull request merged**